packages/ui/src/components/editor/panels/Files/container/index.tsx

```diff
@@ -407,6 +408,32 @@ const FileBrowserContentPanel: React.FC<FileBrowserContentPanelProps> = (props)
       staticResourceModifiedDates.set(modifiedDates)
     }, [staticResourceData.data])
 
+    const handleItemClick = useCallback(
```

Comment:
since this function is used in the same component and is unused in dependency arrays, we should avoid putting it in useCallback as it will be a unnecessary memoization.